ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -83,6 +93,9 @@ internal class PaywallViewModelImpl(
     private val mode: PaywallMode
         get() = options.mode
 
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
```

Comment:
Maybe `PaywallViewModel` can be marked with this instead so we don't add the `@OptIn` in so many methods?